ProjFS.Mac/PrjFSLib/PrjFSLib.xcodeproj/project.pbxproj

```diff
@@ -220,6 +292,35 @@
 /* End PBXSourcesBuildPhase section */
 
 /* Begin XCBuildConfiguration section */
+		4A2C3C5F21BC089A000CF6E1 /* Debug */ = {
```

Comment:
I'm generally quite unhappy with the way the Xcode projects are set up. We're making it unnecessarily hard for ourselves by having 2 independent `xcodeproj`es and trying to paper over the problems with shared schemes.

A better solution would be to put everything into one project. (`ProjFS.Mac/PrjFS.xcodeproj`) If for some reason we really do want multiple sub-projects (possibly because the git commit for the transition would be a mess) we can nest projects instead and create aggregate targets that build multiple sub-targets in the top-level project. I think both approaches would be somewhat out of scope for this PR, although if we want to go ahead with it it may be worth merging that before this PR so we add the daemon on a clean base.

Reply:
I agree with you that we should fold our two separate `xcodeproj` files (for the kext and lib) into one project to make our lives easier. I think this is a design choice that we made early on but as the project has evolved, I don't think it makes sense anymore.

I agree that this is out of scope for this PR, but I'd rather get that change in first so we can add the daemon on a clean base.

Thoughts?

cc: @wilbaker

Reply:
> I agree that this is out of scope for this PR, but I'd rather get that change in first so we can add the daemon on a clean base.

Cleaning this up sounds good to me as well.  I don't have a strong feeling on the ordering of getting this PR and the changes to clean up the projects.  @nickgra, is cleaning up the projects something that you were planning to pick up?

Reply:
@nickgra just filled me on more of the build details offline, and I agree that it makes sense to clean that up with (or before) these changes.  Nick mentioned that he can pick up the task of cleaning up the projects\schemes.

Reply:
@nickgra @pmj outside of the build\configuration changes this PR looks good to me

Reply:
I've now pushed a new version after rebasing on latest master with the new Xcode project, and recreated the log daemon target in there.